drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -73,6 +73,11 @@ def run!(user:)
         processor = container_processor(container)
         raise unknown_field_error(definition) unless processor
 
+        # Check item-level permissions (editor_user_ids) here. The frontend passes all values even if nil, so rather
+        # than raising an exception, just skip processing if we see a value this user doesn't have permission to edit.
+        item = mapped_form_items["#{container}:#{field}"]
+        next if item&.editor_user_ids.present? && !item.editor_user_ids.include?(user.id)
```

Comment:
I think this could be simplified a bit
```suggestion
        next unless item&.editor_user_ids&.include?(user.id)
```

Reply:
It can be simplified a bit, but I think we do still need to check the presence of the array separately, since if the array is not present then we should continue processing. (Everyone can edit items that don't specify `editor_user_ids`)

Reply:
oh right! Thanks